web/packages/teleport/src/Navigation/Navigation.tsx

```diff
@@ -184,6 +185,8 @@ function getTopMenuSection(features: TeleportFeature[]): NavigationSection {
   };
 }
 
+// getNavSubsectionForRoute returns the sidenav subsection that the user is correctly on (based on route).
+// Note that it is possible for this not to return anything, such as in the case where the user is on a page that isn't in the sidenav (eg. Account Settings).
```

Comment:
Should we use `/**`-style comments here for consistency?

Additionally, should the return type be `NavigationSubsection | undefined`?